<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataBase</class>
 <widget class="QMainWindow" name="DataBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataBase</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>4</number>
      </property>
      <widget class="QTreeView" name="treeView">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::TargetMoveAction</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideRight</enum>
       </property>
       <property name="indentation">
        <number>10</number>
       </property>
       <property name="rootIsDecorated">
        <bool>true</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <attribute name="headerHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="headerShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>478</width>
          <height>341</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Edit Activity</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leditActivity">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Date:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit">
             <property name="maximumDate">
              <date>
               <year>2100</year>
               <month>12</month>
               <day>31</day>
              </date>
             </property>
             <property name="minimumDate">
              <date>
               <year>1900</year>
               <month>9</month>
               <day>14</day>
              </date>
             </property>
             <property name="currentSection">
              <enum>QDateTimeEdit::YearSection</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnCalendar">
             <property name="text">
              <string>Open Calendar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Notes:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="notesActivity"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnRemoveActivity">
             <property name="text">
              <string>Remove</string>
             </property>
             <property name="icon">
              <iconset theme="list-remove">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnAddActivity">
             <property name="text">
              <string>Add</string>
             </property>
             <property name="icon">
              <iconset theme="list-add">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>478</width>
          <height>341</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Edit Client</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leditClient">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Year:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QSpinBox" name="spinYearClient">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::PlusMinus</enum>
             </property>
             <property name="correctionMode">
              <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
             </property>
             <property name="minimum">
              <number>1900</number>
             </property>
             <property name="maximum">
              <number>2100</number>
             </property>
             <property name="value">
              <number>1910</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Notes:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="notesClient"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnRemoveClient">
             <property name="text">
              <string>Remove</string>
             </property>
             <property name="icon">
              <iconset theme="list-remove">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnAddClient">
             <property name="text">
              <string>Add</string>
             </property>
             <property name="icon">
              <iconset theme="list-add">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRemove_Selected"/>
   <addaction name="separator"/>
   <addaction name="actionGenerate_Example"/>
  </widget>
  <action name="actionChange_User">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/user_32xLG.png</normaloff>:/Icons/Icons/user_32xLG.png</iconset>
   </property>
   <property name="text">
    <string>Change User</string>
   </property>
  </action>
  <action name="actionRemove_Selected">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/Icons/action_Cancel_16xMD.png</normaloff>:/Icons/Icons/action_Cancel_16xMD.png</iconset>
   </property>
   <property name="text">
    <string>Remove Selected</string>
   </property>
   <property name="toolTip">
    <string>Remove Selected</string>
   </property>
  </action>
  <action name="actionGenerate_Example">
   <property name="text">
    <string>Generate Example</string>
   </property>
  </action>
  <action name="actionWIP">
   <property name="text">
    <string>WIP</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
